doc/doxygen_templates/style.css

```diff
@@ -0,0 +1,132 @@
+/*
```

Comment:
This file will remain the same across all library repos.

Reply:
My intention is not use any custom CSS for a specific library so that all our library docs look the same.